File: LR3/app/src/main/java/com/example/lr3/types/IntegerType.java
package com.example.lr3.types;

import java.io.Serializable;

public class IntegerType implements Serializable {
    private Integer integerValue;

    public IntegerType(Integer integerValue) {
        this.integerValue = integerValue;
    }

    public Integer getIntegerValue() {
        return this.integerValue;
    }

    public void setIntegerValue(Integer integerValue) {
        this.integerValue = integerValue;
    }

    @Override
    public String toString() {
        return String.valueOf(this.integerValue);
    }
}

File: LR2/src/main/kotlin/types/IntegerType.kt
package types

import java.io.Serializable

class IntegerType(private var integerValue: Int) : Serializable {

    fun getIntegerValue(): Int {
        return integerValue
    }

    fun setIntegerValue(integerValue: Int) {
        this.integerValue = integerValue
    }

    override fun toString(): String {
        return integerValue.toString()
    }
}
